audio/libsndfile/Portfile

```diff
@@ -18,36 +19,30 @@ long_description    \
 	a simple, elegant and easy to use API, \
 	and on the fly format conversion.
 license             LGPL-2.1+
-homepage            http://www.mega-nerd.com/libsndfile/
-master_sites        ${homepage}files/
-
+homepage            https://libsndfile.github.io/libsndfile/
```

Comment:
This line shouldn’t be required as the github PG should set this correctly.

Reply:
I'll get rid of that, then.

Reply:
@cjones051073 Would the github PG set it to https://github.com/libsndfile/libsndfile which is just the repo? This was the actual homepage, not the repo.